terra/terra/urls.py

```diff
@@ -126,11 +126,15 @@
     path("", views.index),
 ]
 
-urlpatterns += docs_urls
-urlpatterns += [path("storage/", include("storage.urls"))]
-urlpatterns += [path("tasks/", include("tasks.urls"))]
-urlpatterns += [path("estimators/", include("estimators.urls"))]
-urlpatterns += [path("credits/", include("credits.urls"))]
-urlpatterns += [path("admin/django-rq/", include("django_rq.urls"))]
-urlpatterns += [path("quotas/", include("quotas.urls"))]
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urls_list += docs_urls
+urls_list += [path("storage/", include("storage.urls"))]
+urls_list += [path("tasks/", include("tasks.urls"))]
+urls_list += [path("estimators/", include("estimators.urls"))]
+urls_list += [path("credits/", include("credits.urls"))]
+urls_list += [path("admin/django-rq/", include("django_rq.urls"))]
+urls_list += [path("quotas/", include("quotas.urls"))]
+urls_list += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
This also should be outside the `/v1` prefix, it's used by the admin I think.

Reply:
Ok. You mean line 136 (& 134) or all this 4 lines?

Reply:
Yes sorry, I meant only 136 and 134 (`/admin/django-rq/` and the static endpoint)